esp/services/ws_dfu/ws_dfuService.cpp

```diff
@@ -3488,10 +3504,17 @@ bool CWsDfuEx::doLogicalFileSearch(IEspContext &context, IUserDescriptor* udesc,
     ForEach(*it)
         addToLogicalFileList(it->query(), NULL, version, logicalFiles);
 
-    if (version >= 1.24)
-        resp.setCacheHint(cacheHint);
+    if ((oldCacheHint != cacheHint) && (totalFiles > maxFiles))
```

Comment:
And if that's going to open, oldCacheHint != cacheHint)  will need removing I assume.. correct?

Was the main purpose of this test to prevent it showing on page >=2 ?